modules/cli-manager-custom-index-krew.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-custom-index_{context}"]
+= Adding the CLI Manager custom index to Krew
+
+Use the terminal to add the CLI manager custom index to Krew.
+
+.Prerequisites
+
+* link:https://krew.sigs.k8s.io/docs/user-guide/setup/install[Krew is installed].
+* The CLI Manager is installed.
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+* You are using {product-title} 4.17 or later.
+
+.Procedure
+
+. Execute the following command:
```

Comment:
Should give some detail about what you're doing here.

Also consider using our normal format of "To <x>, run the following command:" or "<x> by running the following command:" for each of your steps throughout this PR

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command